addons/osfstorage/tests/test_views.py

```diff
@@ -432,6 +436,28 @@ def test_archive(self):
     # def test_upload_update_deleted(self):
     #     pass
 
+    def test_add_file_updates_cache(self):
```

Comment:
Nice initial tests, but I feel we need better test coverage for some of these other scenarios - moving into a component/folder, copying into a component/folder, etc.

Reply:
thanks for the additional tests!